src/cityreader/cityreader.py

```diff
@@ -14,20 +19,26 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+import csv
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the 
+    # `cities` list
+    with open('cities.csv', newline='') as csvfile:
+        read_csv = csv.reader(csvfile, delimiter=',')
+        next(read_csv)
+        for row in read_csv:
+            cities.append(City(row[0], float(row[3]), float(row[4])))
     
```

Comment:
Superb usage of `next()` to skip the header as well as casting the lats and lngs to floats.